apps/api/src/music/music.controller.ts

```diff
@@ -32,44 +34,109 @@ export class MusicController {
 
   @Post()
   @ApiOperation({ description: 'Create music' })
+  async create(@Body() data: CreateMusicDto) {
+    const result = await this.musicService.create({
+      ...data,
+    });
```

Comment:
a data has heirarchy won't be cloned perfectly, we can use `Object.assign()` instead.

Reply:
**Objection**

- The spread syntax (...data) is a more modern and concise way of cloning objects. It is part of the ECMAScript 6 (ES6) standard.
- Object.assign({}, data) is an older method that achieves the same result. It is useful in environments where spread syntax might not be supported.